did/plugins/gitlab.py

```diff
@@ -223,7 +233,7 @@ def fetch(self):
             self.user.login, self.options.since, self.options.until,
             'Issue', 'opened')
         self.stats = [
-            Issue(issue, self.parent.gitlab)
+            Issue(issue, self.parent.gitlab, self.parent)
```

Comment:
So if we hand over the whole `parent` now, we could extract the `self.parent.gitlab` in the constructor as well?

Reply:
yes, I'll change that.